CONTRIBUTING.md

```diff
@@ -526,6 +597,59 @@ the original author of the PR is asked if they will submit the relevant
 backport PRs. For a quick guide on how to perform and submit a backport, see 
 the [Backport Guide](Backport-Guide.md) in this repository.
 
+### Forward ports
+
+Forward ports are much less common than backports. However, consolidating a number of standalone repositories into a single repository for
+the Kata 2.0 development effort introduced a potential forward port
+requirement. At the time, both Kata 1.x and 2.0 versions were being developed.
+This means that PRs raised in the Kata 1.x repositories needed to be forward
+ported to the Kata 2.0 repository (since this was to be the next major release
+and should contain all bug fixes and features where possible).
+
+| Initial PR raised | PR type | Backport? | Forward port? |
+|-|-|-|-|
+| Particular 1.x repository | bug fix | stable branches | 2.0 repository |
+| Particular 1.x repository | feature | | 2.0 repository |
+| The 2.0 repository | bug fix | 1.x (and maybe stable branches) | - |
+| The 2.0 repository | feature | | - |
+
+### Porting example
+
+Imagine that you have just raised a new PR on a Kata 1.x repository. The PR
+contains three commits:
+
+- A commit to fix a "typo" in a comment.
+- A commit that changes the way a container is destroyed (and updates the tests).
+- A commit that updates the documentation explaining the new contain
+  destruction behaviour.
+
+Since the PR is not adding any new functionality and since it is correcting
+problems with existing code, it can be considered a bug fix PR. Bug fixes
+should generally be backported. However, this PR was raised in a Kata 1.x
+repository meaning it should also potentially be forward-ported to the Kata
+2.0 repository.
+
+Looking at the tables in the [porting labels](#porting-labels) and the [stable
+branch backports](#stable-branch-backports) sections shows that this PR needs
+to be labelled with the following labels:
+
+- `needs-backport`
+- `needs-forward-port`
+- `stable-candidate`
```

Comment:
@amshinde, @gnawux, @ariel-adam - updating this doc I noticed that we don't use this label, because we haven't created it! :smile: So, do we create it in all repos and require it, or just remove it from this document?

The `stable-candidate` label is *almost* redundant since you can now mark a PR `needs-backport`. But imagine creating a PR in the 2.0 repository - if that is marked `needs-backport`, it's not immediately obvious that the backport may need to go back to Kata 1.x *and* the corresponding stable branches.